exammode/templates/exammode/base.html

```diff
@@ -0,0 +1,217 @@
+{% load i18n %}
+{% load static %}
+{% load base %}
+{% load course %}
+{% load notification %}
+{% load external_services %}
+<!DOCTYPE html>
+<html lang="{{ request.LANGUAGE_CODE }}">
+
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>{% block title %}{% brand_name %}{% endblock %}</title>
+
+	<script src="https://code.jquery.com/jquery-1.12.4.min.js" integrity="sha384-nvAa0+6Qg9clwYCGGPpDQLVpLNn0fRaROjHqs13t4Ggj3Ez50XnGQqc/r8MhnRDZ"
+	 crossorigin="anonymous"></script>
+
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet" />
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+
+	<link href="https://use.fontawesome.com/releases/v5.3.1/css/solid.css" rel="stylesheet" integrity="sha384-VGP9aw4WtGH/uPAOseYxZ+Vz/vaTb1ehm1bwx92Fm8dTrE+3boLfF1SpAtB1z7HW"
+	 crossorigin="anonymous">
+	<link href="https://use.fontawesome.com/releases/v5.3.1/css/regular.css" rel="stylesheet" integrity="sha384-ZlNfXjxAqKFWCwMwQFGhmMh3i89dWDnaFU2/VZg9CvsMGA7hXHQsPIqS+JIAmgEq"
+	 crossorigin="anonymous">
+
+	<link href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.7/styles/github.min.css" rel="stylesheet" />
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.6/highlight.min.js"></script>
+
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.5.12/clipboard.min.js"></script>
+
+	<script src="{% static 'js/translate.min.js' %}"></script>
+
+	{% if is_teacher %}
+	<link data-translation rel="preload" as="fetch" hreflang="fi" href="{{ STATIC_URL }}js-translations/teacher.fi.json">
```

Comment:
Templates should use the `{% static %}` template tag in order to link to static files: https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#static

Reply:
Add js and css libraries to assets folder

Update jquery to a newer version 3.x